charts/newrelic-infrastructure-beta/values.yaml

```diff
@@ -0,0 +1,398 @@
+            matchNode: true
+            endpoints:
+              - url: https://localhost:8443
+                insecureSkipVerify: true
+                auth:
+                  type: bearer
+              - url: http://localhost:8080
+          - selector: "k8s-app=kube-apiserver"
+            namespace: kube-system
+            matchNode: true
+            endpoints:
+              - url: https://localhost:8443
+                insecureSkipVerify: true
+                auth:
+                  type: bearer
+              - url: http://localhost:8080
+          - selector: "app=openshift-kube-apiserver,apiserver=true"
+            namespace: kube-system
+            matchNode: true
+            endpoints:
+              - url: https://localhost:8443
+                insecureSkipVerify: true
+                auth:
+                  type: bearer
+
+kubelet:
+  enabled: true
+  annotations: {}
+  tolerations:
+    - operator: "Exists"
+      effect: "NoSchedule"
+    - operator: "Exists"
+      effect: "NoExecute"
+  nodeSelector: {}
+  affinity:
+    nodeAffinity: []
+  extraEnv: []
+  extraEnvFrom: []
+  extraVolumes: []
+  extraVolumeMounts: []
+  resources:
+    limits:
+      memory: 300M
+    requests:
+      cpu: 100m
+      memory: 150M
+  scraperConfig:
+    kubelet: {}
+
+
+ksm:
+  enabled: true
+  annotations: {}
+  tolerations:
+    - operator: "Exists"
+      effect: "NoSchedule"
+    - operator: "Exists"
+      effect: "NoExecute"
+  nodeSelector: {}
+  affinity:
+    podAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            topologyKey: kubernetes.io/hostname
+            labelSelector:
+              matchLabels:
+                app.kubernetes.io/name: kube-state-metrics
+          weight: 100
+    nodeAffinity: []
+  # Extra env ‘items’ that will be added to the definition of the containers; a string is expected (can be templated).
+  extraEnv:
+    - name: ENV_VAR1
+      value: "var1"
+    - name: ENV_VAR2
+      value: "var2"
+  # Extra envFrom ‘items’ that will be added to the definition of the containers; a string is expected (can be templated).
+  # extraEnvFrom:
+  #  - configMapRef:
+  #      name: 'another-config-map'
+  extraEnvFrom: []
+  # Mount additional volumes into the pod.
+  extraVolumes: []
+  # Mount additional volumes into the pod.
+  extraVolumeMounts: []
+  resources:
+    limits:
+      memory: 300M
+    requests:
+      cpu: 100m
+      memory: 150M
+  scraperConfig:
+#    interval: 10s
+#    timeout: 40s
+    ksm:
+      namespace: ""
+      port: 0
+      scheme: ""
+      selector: ""
+      staticURL: ""
+
+common:
+  scraperConfig:
+    interval: 15s
+    timeout: 30s
+
+
+# K8s DaemonSets update strategy.
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxUnavailable: 1
+
+# Kubelet's agent envs
+# Custom attributes to be passed to the New Relic agent
+customAttributes:
+  clusterName: $(CLUSTER_NAME)
```

Comment:
Customers could delete this by accident, and we need it for integrations to report sensical things. Would it make sense to add the special `clusterName` key with a helper template?

`customAttributes: merge .Values.customAttributes (dict "clusterName" "$(CLUSTER_NAME)")`

Or something like that

Reply:
Good point.